packages/stress-testing/Dockerfile

```diff
@@ -41,5 +41,6 @@ RUN yarn
 RUN yarn csb:build
 
 ENV MONOREPO_ROOT=/monorepo
+ENV SINGLE_LINE_LOGS="true"
```

Comment:
can you set this in `packages/stress/scripts/start.sh` next to `DEBUG_DEPTH`?

Reply:
sure

Reply:
do you want me to remove from the dockerfile?